File: leetcode-questions/java/src/test/java/com/leetcode/questions/removeduplicatesfromsortedarray/RemoveDuplicatesFromSortedArrayTest.java
package com.leetcode.questions.removeduplicatesfromsortedarray;

import com.leetcode.questions.leetcode.editor.en.RemoveDuplicatesFromSortedArray;
import com.leetcode.questions.support.GeneratedLeetCodeTestHarness;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
import java.nio.file.Files;
import java.nio.file.Path;

public class RemoveDuplicatesFromSortedArrayTest {

    private static final Path CASES_PATH = Path.of("generated-test-cases", "remove-duplicates-from-sorted-array.json");

    public RemoveDuplicatesFromSortedArrayTest() {

    }

    @Test
    public void testCases() throws Exception {
        GeneratedLeetCodeTestHarness.run(RemoveDuplicatesFromSortedArray.Solution.class, "removeDuplicates", Files.readString(CASES_PATH));
    }
}

File: leetcode-questions/csharp/src/test/csharp/com/leetcode/questions/removeduplicatesfromsortedarray/RemoveDuplicatesFromSortedArrayTest.cs
using Xunit;
using System.Collections.Generic;
using com.leetcode.questions.leetcode.editor.en.removeduplicatesfromsortedarray;
using com.leetcode.questions.support;

namespace com.leetcode.questions.removeduplicatesfromsortedarray;

public class RemoveDuplicatesFromSortedArrayTest {

    private const string CasesFileName = "remove-duplicates-from-sorted-array.json";

    public RemoveDuplicatesFromSortedArrayTest() {

    }

    [Fact]
    public void TestCases() {
        GeneratedLeetCodeTestHarness.Run(typeof(Solution), "RemoveDuplicates", GeneratedLeetCodeTestHarness.ReadCasesFile(CasesFileName));
    }
}
